blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -116,4 +118,4 @@ PipelinePage.childContextTypes = {
 const selectors = createSelector([pipelineSelector],
     (pipeline) => ({ pipeline }));
 
-export default connect(selectors, actions)(documentTitle(PipelinePage));
+export default translate(['jenkins.plugins.blueocean.dashboard.Messages'], { wait: true })(connect(selectors, actions)(documentTitle(PipelinePage)));
```

Comment:
The syntax used in core JS seems very simple and easy to understand. Is there a reason we can't just use that in the rest of the app, or am I misunderstanding something?